INSTALL

```diff
@@ -82,7 +82,20 @@
     $ make
     $ make test
     $ make install
-
+    
+  on Mac OS/X compile for iOS (eg. arm64/armv7)
+  
+    $ ./Configure iphoneos-cross --prefix=/Users/hongs/openssl-1.1.0c/openssl_arm64
+    
+    then, change CC="..." into CC="xcrun -sdk iphoneos clang -mios-version-min=7.0 -arch arm64” and delete "-isysroot $(CROSS_TOP)/SDKs/$(CROSS_SDK)" of CFLAGS in Makefile.
```

Comment:
Absolutely not. I mean there is no place for that kind of instructions in our notes. If anything, xcrun deserves own targets. See #6410. Also note that iphoneos-cross is not actually proper target for 64-bit.  Not to mention that it doesn't engage assembly support. It's actually deprecated. ios-cross and ios64-cross are ones to use. Well, ios-xcrun and ios64-xcrun once #6410 is merged. Is there is desire/need to add iOS-specific notes, then 15-ios.conf would be better place. Unless one has enough words for separate NOTES.iOS...

Reply:
I would rather see a NOTES file, if they are meant to be read by our users